python_modules/dagster/dagster/_core/execution/plan/execute_step.py

```diff
@@ -497,28 +496,6 @@ def _get_output_asset_materializations(
         tags[BACKFILL_ID_TAG] = backfill_id
 
     if asset_partitions:
-        metadata_mapping: Dict[
```

Comment:
are these metadata changes supposed to be in this PR or is this a stacking artifact?